.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "benchmarks/tpch/tpch-dbgen"]
```

Comment:
I wanted to avoid using submodules, but C libraries are kinda a PITA to work with, so this seemed to be the only manageable way of importing this without copy/pasting the entire repo into our benchmarks.

Reply:
I think that's fine. If you're not running benchmarks, then no need to init the submodule.

My concern with this however is generating the the data can be pretty slow. I didn't read through all of this to see if we're downloading data we've generated or not, so unsure if this is valid concern.

Reply:
> My concern with this however is generating the the data can be pretty slow. I didn't read through all of this to see if we're downloading data we've generated or not, so unsure if this is valid concern.

generating the data is pretty slow, especially on larger scale factors, but this seemed like the most reproducible way without having to manage the datasets in s3 or gcs. They are all created on the spot.

Reply:
the options seem like:
- submodules may be fine, actually
- `git clone` in a justfile target. (basically a sub-module except there aren't top-level references for it, and may be easier to test out?) 
- write a script to generate and upload files to GCS, and run it in CI when one of the input files changes.

Reply:
> write a script to generate and upload files to GCS, and run it in CI when one of the input files changes.

I wasn't expecting us to run these benches in the CI, but instead ad-hoc

Reply:
how long do the benchmarks take to run? 

I think, even if we don't run them on every commit or even every day (as in a timer, that we use the CI infrastructure as a (potentially) more stable way to run these? There's going to be a lot of variability between peoples machines and setups, and if we detect a regression it's easier to find if we can do the binary search in parallel, at least?

Reply:
I opened up https://github.com/GlareDB/glaredb/issues/1822 thinking we could just run them on every merge into main, with a workload size of one (and each merge would queue up benchmark runs). I think this would get us the data we want at a pretty granular level without blocking anyone's workflow.

Reply:
> I opened up https://github.com/GlareDB/glaredb/issues/1822 thinking we could just run them on every merge into main.

That sounds reasonable. scale of 1 should be pretty quick to generate & run. (<5 min).